docker-compose-rule-core/src/main/java/com/palantir/docker/compose/DockerComposeManager.java

```diff
@@ -281,47 +281,18 @@ public String run(DockerComposeRunOption options, String containerName,
         return dockerCompose().run(options, containerName, arguments);
     }
 
-    public interface BuilderExtensions {
-        BuilderExtensions file(String dockerComposeYmlFile);
+    public interface BuilderExtensions<B> {
```

Comment:
nit: type T instead of B?

Reply:
B for Builder was what I was going for :p

Reply:
TSelf might actually be a better description of what it is